keyboards/crkbd/keymaps/copface/config.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#define LEADER_TIMEOUT 300
+#define TAPPING_TERM 200
+// #define IGNORE_MOD_TAP_INTERRUPT
+// #define PERMISSIVE_HOLD
+// #define TAPPING_FORCE_HOLD
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This is the default behavior, and should be removed (it no longer does anything).
```suggestion
```